### ISSUE TITLE
segmentation training result

### ISSUE THREAD

**Issue by LiYuxian0306** [andyhandsom6/ExCap_3D_Pro#1]
Thank you for providing your own reproduction code for excap3d and the improvements you have made! 

I encountered some difficulties during the segmentation training process. The AP value of the validation set has always been very low, and Mask3D has not opened the model weights on the scannetpp dataset either.

Therefore, I hope to ask whether you could generously provide some of your results and settings of segmentation training (AP rate, augmentation settings, loss, training steps used, and so forth), or may I ask how you find this segmentation performs?

Thank you so much!
